core/templates/dev/head/pages/exploration_editor/EditorModeServiceSpec.js

```diff
@@ -16,25 +16,25 @@
  * @fileoverview Unit tests for the EditorModeService.js
  */
 describe('Editor Mode Service', function() {
-  var service;
+  var srv;
```

Comment:
Yep, editorModeService is fine. I was actually thinking about `ems` (similar to what we do in EditorServices.js).

That said, I see where you got the convention from -- it looks like @ashish3805 had gone with `srv` in #3091 and I didn't notice it. So I can see why this conversation might have been confusing, sorry about that. The general convention in this codebase is to use an abbreviation of the service like `ems`, but on reflection it is probably better for the convention to be what you've finally settled on, i.e. `editorModeService`.

CC-ing @ashish3805 so that he can take note of this for future tests.